### ISSUE TITLE
PurgeCSS is misconfigured for `:global(...)`

### ISSUE THREAD

**Issue by babichjacob** [babichjacob/sapper-postcss-template#18]
Reproduction:
1. Add `:global(body) { background-color: green; }` in any normal `style` block
2. Run development (where PurgeCSS doesn't run) and see that the body background *is* green
3. Run production (where PurgeCSS runs) and see that the body background is unstyled

My diagnosis:
PurgeCSS is misconfigured and not safelisting `body`, because when it saw `:global(body)` it interpreted it as there being a `:global(body)` selector rather than a `body` selector it needed to see to allow that styling to stay in

**Comment by babichjacob** [comment]
Workaround:
Make the whole style block global:
```html
<!-- lang="postcss" is implicitly set because it's one of the `defaults` in `svelte-preprocess` config -->
<style global>
    body {
        background-color: green;
    }
</style>
```

**Comment by babichjacob** [comment]
Closed by 6b596196a8b052e1a65ca59bc2b8fcad61e10e8d
